frontend/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 100
```

Comment:
The printWidth is slightly different to the eslint rule. It's the preferred length, vs a max limit with eslint. More details here: https://prettier.io/docs/en/options.html#print-width

Reply:
> To make sure eslint would not fix prettier changes, I tried this:
> 
> ```
> $ ./node_modules/.bin/prettier --write src
> # 2800 or so changed lines
> $ git commit . -m "prettier changes"
> $ yarn lint --fix src
> $ git status
> On branch prettier
> nothing to commit, working tree clean
> ```
> 
> We can either
> 
> 1. make one big code formatting commit
> 2. do it as we go, and only format files as they change
> 
> Probably 2 is the safest option, with less noise too. If we went with 1 big commit, there's some trick you can do to get git blame working again.

I am not sure whether step 2 will be good here because otherwise than each time somebody is doing a different pr it will bring in the noise of formatting there. It's best if we have the formatting done here in a separate commit.
let's see what @joaquimrocha has to say.

Reply:
Yes, I think joaquimrocha has this view too. Let's go with 1. make one big code formatting commit.